src/main/java/de/muenchen/isi/domain/service/AbfrageService.java

```diff
@@ -192,12 +193,14 @@ public InfrastrukturabfrageModel patchAbfrageInBearbeitungFachreferate(
     public InfrastrukturabfrageModel changeStatusAbfrage(
         final UUID id,
         final StatusAbfrage statusAbfrage,
-        final String anmerkung
+        final String anmerkung,
```

Comment:
Die Attribute anmerkung und schnellesSchließenAbfrage sind in den Javadocs zu dokumentieren.